src/test-private.c

```diff
@@ -380,7 +380,7 @@ static void test_unquote_double(void) {
         c_assert(!memcmp(buf, "fo\"obar", strlen(string) - 3));
 }
 
-int main(int argc, char *argv[]) {
+int main(__attribute__((__unused__)) int argc, __attribute__((__unused__)) char *argv[]) {
```

Comment:
Can you just change this to `int main(void) {` instead? No reason to specify argc/argv when they are not used.

Reply:
Yes! Actually we could do this under c-stdaux as well, what do you think about that?

Reply:
As for this project tests, it should be good now; I'll mark this as resolved

Reply:
Yeah, I wouldn't mind adjusting the others as well! Thanks a lot for the update!

Reply:
No problem, I'll do that today/tomorrow.